src/test/kotlin/sirius/web/http/CSRFTokenTest.kt

```diff
@@ -0,0 +1,181 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.web.http
+
+import io.netty.handler.codec.http.HttpHeaderNames
+import io.netty.handler.codec.http.HttpResponseStatus
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import sirius.kernel.SiriusExtension
+import sirius.kernel.commons.Streams
+import java.net.HttpURLConnection
+import java.net.URL
+
+import java.nio.charset.StandardCharsets
+import kotlin.test.assertEquals
+
+@ExtendWith(SiriusExtension::class)
+class CSRFTokenTest {
+
+    @Test
+    fun `safePOST() passes test if token is missing via GET`() {
```

Comment:
This is misleading. Actually safePOST() does not pass. But every test should pass, so providing that information is not relevant. 
The whole setup in here isn't to elegant, e.g. we do blacklist tests on web api level and test description mentions internal method that isn't visible in test.
The test endpoint expecting valid csrf token should also not be called "fake-delete-data" but maybe "csrf-protected". But works for me like this. I don't won't to slow down kotlin migration to much